tests/cypress/support/ui.ts

```diff
@@ -14,3 +14,5 @@ export const oneTabletViewport = () =>
 
 export const oneMobileOrTabletViewport = () =>
   [randomInt(MIN_WIDTH, DESKTOP_BREAKPOINT), randomInt(MIN_HEIGHT, MAX_HEIGHT)] as const
+
+export const allViewports = () => [oneDesktopViewport(), oneMobileViewport(), oneTabletViewport()] as const
```

Comment:
Testing everything with all viewports is generally wasteful, i.e. unnecessarily increasing test times. IMO unless the test has viewport dependencies, testing with one random viewport is good enough to catch edge cases

Reply:
This is what I'm using to test the new llamalend tables: https://github.com/curvefi/curve-frontend/blob/c9530c0b13a2e3a63b784f893d5ddffe64f0354d/tests/cypress/support/ui.ts#L19